src/Subscriber/OrderStateChangeSubscriber.php

```diff
@@ -89,6 +89,12 @@ public function onOrderDeliveryStateReturned(OrderStateMachineStateChangeEvent $
 
             $altaPayTransaction = $this->getAltaPayTransaction($order);
 
+            $refundedAmount = (float)$altaPayTransaction->RefundedAmount;
+            if ($refundedAmount > 0) {
```

Comment:
Is this check correct? If the refundedAmount is more than 0, doesn't that mean that the refund has been processed ?